.github/workflows/release.yaml

```diff
@@ -29,3 +29,8 @@ jobs:
       env:
         GH_WRITE_TOKEN: ${{ secrets.GH_WRITE_TOKEN }}
       run: ./.ci/create-release-github.sh
+
+    - name: "refresh go proxy module info on release"
+      run: |
+        OPERATOR_VERSION=$(git describe --tags)
+        curl https://proxy.golang.org/github.com/open-telemetry/opentelemetry-operator/@v/${OPERATOR_VERSION}.info
```

Comment:
How about running `go get` or `get install` to fetch the new version instead?

Reply:
I am not sure how to fetch the latest version using `go get` do you mind sharing the cmd to fetch the version? Currently, I am using the same method used to create release tag which is already being used in prior step of release flow [here](https://github.com/open-telemetry/opentelemetry-operator/blob/main/.ci/create-release-github.sh#L3)

Reply:
```
go get github.com/open-telemetry/opentelemetry-operator@{VERSION}
```

should work. For example,

```
go get github.com/open-telemetry/opentelemetry-operator@v0.27.0
```

Reply:
Did it work to use the notation that @owais mentioned?

Reply:
the `go get` just pulls the dep, but I am unable to test the refresh thing as it happens after the releases. How about letting it be `curl` as suggested in go docs?

Reply:
I think `curl` is more explicit about the intentions. It would also remove the doubts on whether the call actually got to the destination. I was referring to the env var with the version, though :-)

Reply:
Switched it back to `curl` and I noticed during the release process we are creating operator release in `create-release-github.sh` using the same git cmd [here](https://github.com/open-telemetry/opentelemetry-operator/blob/main/.ci/create-release-github.sh#L3) so now during the release we create a new version and perform the curl on the same version with current changes.